### PR TITLE
The Next Web Project

### PR DESCRIPTION
**Pull request by Gnwin**
On this project, we learnt about responsive design. Here, we implemented:

CSS grid layouts
Float 
Flex

### CODE REVIEW

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Hi, good job so far guys :clap: :clap: :clap:, but you still have to make some changes :toolbox: in order to get this project approved :checkered_flag: 

- [x] colors are really important please use the same colors on the original page.

- [x] you forgot to work on the desktop view, cause you supposed to have three breaking points mobile, tablet, and desktop.

- [x] on tablet view you should remove the tnw logo from the hero section:

image

- [x] Fix the text on the hero section they are not displayed correctly:
image

- [x] On mobile view remove the yellow section form the header and fix the search icon also add a burger menu like in the original:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] on the latest news section please fix the images on the mobile view:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] use different images on your project so it looks more professional.

- [x] align article titles with the images, like in the original:

|*Clone*|*Original*|
|:----:|:----:|
| image | image |

- [x] make sure that these type of articles are matching, they should have the same layout, typography, and colors:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] add a title to each of the previous mentioned articles.

- [x] Remove this title:
image

- [x] you forgot to add latest deals section please add one:
image

- [x] Fix linter errors:
image

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

**Comment by Gnwin** [comment]
I was able to correct all changes from my last TSE review. I worked on this project alone after I was de-partnered. Awaiting approval.

File: assets/stylesheets/style.css
Comment (descholar-ceo) [inline assets/stylesheets/style.css]: Do you really need this comments? If so, try to make them descriptive, so that in the future you will be able to understand why you put them here, or other fellow programmers will understand what you meant with them.

File: assets/stylesheets/style.css
Comment (descholar-ceo) [inline assets/stylesheets/style.css:940]: Since the project specifications requires you to go mobile first approach, you should try to make this a mobile first approach as well, [see the screenshot of that requirement here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png)

File: README.md
Comment (descholar-ceo) [inline README.md]: Since the readme file will be the official document on the default branch of your project, there is no need to talk about your `nextweb branch`, it will not stay there for ever, after your PR is merged to the main branch, then there will be no need to checkout that `nextweb branch`. So, I highly recommend you to remove this `nextweb branch` related information from your readme file

File: README.md
Comment (descholar-ceo) [inline README.md]: If installing linters is a requirement to be able to check your project locally (on one's computer), please elaborate those linters that are needed. I mean list them hiere.

File: README.md
Comment (descholar-ceo) [inline README.md:59]: If you have more social links such as `linkedin` or `twitter`, you should have added them here as well.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @Gnwin :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] Instead of saying latest news, you should say latest deals (:point_right: This was requested in the previous review, where a TSE suggested you to add the `latest deal` section)

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-10-25 10-19-56 | Screenshot from 2020-10-25 10-20-43 |
:point_right: Could you see the difference on titles :question: :question: :question: 

- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [x] In the navbar, you should add the small dropdown icons on right side `Events` and `Business` options in the leftside.


| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-10-25 10-35-42 | Screenshot from 2020-10-25 10-35-03 |

:point_right: Could you see the difference :question: :question: :question: 

- [x] The breakpoints: 767px, 768px, 769px; you should just combine them in one breakpoint, they are almost the same.

- [x] Try to make your classes more descriptive, some of your classes are `pb, in, fs1, rig, sf, bc ...` to name a few, those and others like them, you should try to make them more descriptive

- [x] I liked that you have used sementic HTML, but I recommend you to try use more, like using `article` tags for the articles

- [x] Your commit messages should be in imperative mode or present tense, you should have used `Finish with first breakpoint(767px)` instead of `finishing with first breakpoint(767px)`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: assets/stylesheets/style.css
Comment (Gnwin) [reply assets/stylesheets/style.css]: Done

**Comment by Gnwin** [comment]
Thank you so much for your prompt and detailed reply. I have implemented all changes except the one I indicated below.

> The breakpoints: 767px, 768px, 769px; you should just combine them in one breakpoint, they are almost the same.

The breakpoints are all unique and performing different functions. Some are min-width and max-width, therefore doing so will affect the views.

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
## :clap: :clap: :clap: Congratulations @Gnwin, your project is approved :tada: :tada: :tada:
You have implemented all of the minum requirements of this project. You can go ahead and merge the changes on the base branch, and remember to submit a mandatory project completion form,

See you in next projects, :traffic_light: :traffic_light: :traffic_light: 

### Happy Codding :computer: :computer: :computer:

[Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)
[LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

**Comment by Gnwin** [comment]
Thank you so much.

**Pull request merged**
